packages/next/src/client/components/app-router-headers.ts

```diff
@@ -14,3 +14,5 @@ export const FLIGHT_PARAMETERS = [
   [NEXT_ROUTER_STATE_TREE],
   [NEXT_ROUTER_PREFETCH],
 ] as const
+
+export const NEXT_RSC_UNION_QUERY = '__nextRSC' as const
```

Comment:
This one should be kept as short as possible
```suggestion
export const NEXT_RSC_UNION_QUERY = '_rsc' as const
```